ingress/controllers/nginx/README.md

```diff
@@ -196,6 +196,16 @@ Use the [custom-template](examples/custom-template/README.md) example as a guide
 **Please note the template is tied to the go code. Be sure to no change names in the variable `$cfg`**
 
 
+### Authentication
+
+Is possible to add authentication adding additional annotations in the Ingress rule. The source of the authentication is a secret that contains usernames and passwords inside the the key `auth`
+
+The annotations are:
+```
+ingress-nginx.kubernetes.io/auth-type:[basic|digest]
+ingress-nginx.kubernetes.io/auth-secret:secretName
+ingress-nginx.kubernetes.io/auth-realm:"realm string"
```

Comment:
Please give an example of the secret too. What is realm, basic,digest? (or is there a doc you can point at that explains them), or link to your example below.